onnxruntime/python/tools/transformers/onnx_model.py

```diff
@@ -426,6 +426,60 @@ def find_first_child_by_type(self, node, child_type, input_name_to_nodes=None, r
 
         return None
 
+    def match_child_path(
+        self,
+        node,
+        child_op_types,
+        child_output_index=None,
+        output_name_to_node=None,
```

Comment:
Are you planning to use `output_name_to_node` for something? It is not currently used.